Classes/Executor/OptimizationExecutorRemoteImageoptim.php

```diff
@@ -30,21 +30,105 @@
 
 class OptimizationExecutorRemoteImageoptim extends OptimizationExecutorRemote
 {
+
     /**
-     * Optimize image using remote Imageoptim
-     * Return the temporary file path
+     * Optimize image
      *
-     * @param string $inputImageAbsolutePath Absolute path/file with image to be optimized. It will be replaced with optimized version.
-     * @param Configurator $configurator Executor configurator
-     * @return ExecutorResult Executor Result
+     * @param $inputImageAbsolutePath string Absolute path/file with image to be optimized
```

Comment:
Why first $inputImageAbsolutePath and then type "string" and not reversed like two annotations below?